CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # node-google-dfp-wrapper
 
+### 1.3.0
+- Adds `getCriteriaValues` for querying criteria values without extracting id.
+- Adds `getCriteriaValueId` for querying criteria values and extracting id.
```

Comment:
Maybe `getCriteriaValuesId`?  I'm not sure what the roll of plural vs non plural is with these 3 commands.  I would expect the deprecated `getCriteriaValue` to get one value, but it looks like it gets multiple.  That's fine because it's deprecated.  For the new methods, I would expect `getCriteriaValueId` to get a single value with ID and `getCriteriaValues` to get multiple values without ID.